### PR TITLE
[Bugfix:Developer] Fix abandoned tagger & remove warning

### LINKED ISSUE #10609: Abandoned PR auto-tagger is broken
**Describe the bug**
The abandoned PR auto-tag action appears to be broken *again*.  Despite new activity 3 days ago, the action added the abandoned tag [here](https://github.com/Submitty/Submitty/pull/9906#event-13258588125).

image

**Expected behavior**
This action should function as expected 100% of the time.  What can we do to ensure the reliability of this action going forward?  This is at least the 3rd time the action has been broken since it was originally added.  See https://github.com/Submitty/Submitty/issues/10449 for the previous issue I created about this topic.

### PR DESCRIPTION
**Pull request by powe97**
### What is the current behavior?
Fixes #10609. Currently, the abandoned PR tagger checks if a PR has been already given a warning and has had two days of inactivity. When an abandoned PR is taken over, the tagger will re-apply the abandoned tag after 2 days instead of 14 days of inactivity. This is undesirable.

### What is the new behavior?
This PR removes the warning message and simply has the script tag a PR after 14 days of inactivity.

### CODE REVIEW

**Comment by sahilsuman933** [comment]
LGTM :rocket:

**Review by sahilsuman933 — APPROVED** [review]

File: .github/bin/label_abandoned_prs.py [hunk 1]
```diff
@@ -7,45 +7,45 @@
...
     already_abandoned = False
-    for labels in json_data['labels']:
-        if labels['name'] == 'Abandoned PR - Needs New Owner':
+    for labels in json_data["labels"]:
+        if labels["name"] == "Abandoned PR - Needs New Owner":
             already_abandoned = True
 
     approved = False
-    for review in json_data['reviews']:
+    for review in json_data["reviews"]:
```
Comment (williamjallen) [inline .github/bin/label_abandoned_prs.py:31]: Please limit this PR to only the most minimal set of changes necessary.  If you wish to make formatting changes, please make them in a separate PR so formatting changes don't hide logic changes.

**Review by williamjallen — CHANGES_REQUESTED** [review]
The changes seem reasonable overall, but the formatting changes need to be reverted before this can be merged.

File: .github/bin/label_abandoned_prs.py (hunk 1 above)
Comment (powe97) [reply .github/bin/label_abandoned_prs.py:31]: Ah oops. I ran an auto-formatter on that, I can revert that.

Comment (powe97) [reply .github/bin/label_abandoned_prs.py]: Reverted.

**Review by micpap25 — APPROVED** [review]

**Review by williamjallen — APPROVED** [review]
Changes look reasonable to me.
